ramos/mixins.py

```diff
@@ -24,12 +24,12 @@ class SingletonCreateMixin:
     _instances = {}
 
     @classmethod
-    def create(cls):
+    def create(cls, *args, **kwargs):
         """
         Return always the same instance of the backend class
         """
         if cls not in cls._instances:
-            cls._instances[cls] = cls()
+            cls._instances[cls] = cls(*args, **kwargs)
```

Comment:
As mentioned by @jarussi, I think this behavior would be misleading. Suppose I have a backend created with:
```python
CacheBackend.create(type='memcached')
```

If it inherits from `SingletonCreateMixin`, the next time I could call
```python
CacheBackend.create(type='redis')
```

And get the instance with `type='memcached'` (because it was created first). 

You could use the `*args` and `**kwargs` in the key for the singleton cache, but all parameters would have to be hashable. Another option would be calling `hash()` at the backend instance and use it as the singleton key. That would require the Singleton to implement `__hash__` properly (otherwise, the Singleton would not be a singleton).

Reply:
Thanks for the review guys. Looking now, the behavior in Singleton could be affected by this changes, makes a lot of sense. The main problem here is that method `create` from SingletonCreateMixin is incompatible with initialization present in [Pool](https://github.com/luizalabs/ramos/blob/main/ramos/pool.py#L18). I will apply a hash using arguments like @renanivo suggested to prevent initialization mistakes. Thanks!